### PR TITLE
Remove Jetpack Pricing Asterisks

### PR DESCRIPTION
**Pull request by cbauerman**
#### Changes proposed in this Pull Request

* Remove all asterisks from Jetpack Pricing Pages
* Remove asterisk footnotes

| Jetpack Pricing Page (Top) | Jetpack Pricing Page (Bottom) | Storage Upgrade Page |
| -------------------------- | ----------------------------- | -------------------- |
| Screen Shot 2022-02-15 at 09 04 19 | Screen Shot 2022-02-15 at 09 04 49 | Screen Shot 2022-02-15 at 09 03 43 |

#### Testing instructions

1. Boot Branch or use Live Link as Calypso Green.
2. Navigate to `/pricing`.
3. Verify that the following asterisks have been removed:
   - The Intro Pricing Banner
   - The Discount Labels on each Pricing Card
4. Verify the footnotes at the bottom of the page ( below the FAQ ) is gone
5. Boot Branch or use Live Link as Calypso Blue.
6. Navigate to `/plans/storage/:siteId` for a site with Backup T1 or Security T1.
7. Verify that the Asterisks on the "save 50% off" monthly/annual toggle is gone.
8. Verify the footnotes at the bottom of the page ( below the FAQ ) are gone.

Related to 1196108640073826-as-1201681966634228

### CODE REVIEW

File: client/components/jetpack/intro-pricing-banner/index.tsx [hunk 1]
```diff
@@ -67,11 +67,10 @@ const IntroPricingBanner: FunctionComponent = () => {
 					/>
 					<span>
 						{ preventWidows(
-							translate( 'Get %(percent)d%% off your first year.*', {
+							translate( 'Get %(percent)d%% off your first year.', {
```
Comment (a8ci18n) [inline client/components/jetpack/intro-pricing-banner/index.tsx:70]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Get %(percent)d%% off your first year*' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/remove/jetpack-pricing-asterisks)

File: client/components/jetpack/intro-pricing-banner/index.tsx [hunk 2]
```diff
@@ -87,9 +86,8 @@ const IntroPricingBanner: FunctionComponent = () => {
 						/>
 						<span>
 							{ preventWidows(
-								translate( '%(days)d day money back guarantee.**', {
+								translate( '%(days)d day money back guarantee.', {
```
Comment (a8ci18n) [inline client/components/jetpack/intro-pricing-banner/index.tsx:89]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/plans/jetpack-plans/plans-filter-bar/index.tsx [hunk 3]
```diff
@@ -42,7 +42,7 @@ const DiscountMessage: React.FC< DiscountMessageProps > = ( { toggleChecked } )
 								args: { discount: INTRO_PRICING_DISCOUNT_PERCENTAGE },
 								comment: 'Discount is a percentage',
 						  } )
-						: translate( 'Get %(discount)s%% off*', {
+						: translate( 'Get %(discount)s%% off', {
```
Comment (a8ci18n) [inline client/my-sites/plans/jetpack-plans/plans-filter-bar/index.tsx:45]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

**Review by manzoorwanijk — APPROVED** [review]
Tested and it works as expected.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7143519

Thank you @cbauerman for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7143519).

**Pull request merged**